front/src/components/mmt-user-menu/mmt-user-menu.component.css

```diff
@@ -29,10 +29,6 @@
       color: #B6B6B6;
```

Comment:
it is not your changes, but why is it on in css variables

Reply:
fixed